garak/resources/beast/beast_attack.py

```diff
@@ -0,0 +1,403 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Union, Optional, Tuple
+
+import torch
+import numpy as np
+
+import garak._config
+from garak.generators import Generator
+from garak.resources.common import REJECTION_STRINGS, load_advbench
+from tqdm import tqdm
+
+# System prompt pulled from reference implementation
+SYSTEM_PROMPT = (
+    "A chat between a curious user and an artificial intelligence assistant. "
+    "The assistant gives helpful, detailed, and polite answers to the user's questions."
+)
+beast_resource_data = garak._config.transient.basedir / "resources" / "beast" / "data"
```

Comment:
```suggestion
resource_beast_data = garak._config.transient.basedir / "resources" / "beast" / "data"
```

Nitpick, it seems reasonable to have the name match the path is selects. May become a moot point depending on long term location for staging file.

Reply:
This var name reads weirdly to me, but I get it. I think once we get #479 squared away, it's likely a moot point.